src/util/math.js

```diff
@@ -11,8 +11,8 @@ const { Uncertainty } = require('../datatypes/uncertainty');
 
 const MAX_INT_VALUE = Math.pow(2, 31) - 1;
 const MIN_INT_VALUE = Math.pow(-2, 31);
-const MAX_FLOAT_VALUE = 99999999999999999999999999999.99999999;
-const MIN_FLOAT_VALUE = -99999999999999999999999999999.99999999;
+const MAX_FLOAT_VALUE = 9999999999999999999999999999.99999999;
+const MIN_FLOAT_VALUE = -9999999999999999999999999999.99999999;
```

Comment:
So, here's a plot twist.  In looking into this further for this PR, it seems that:
* Yes, the CQL execution values were _wrong_
* BUT, the spec test values are _also wrong_!

According to [CQL Spec](http://build.fhir.org/ig/HL7/cql/09-b-cqlreference.html#maximum):
* Max: `(10^28 – 1) / 10^8`  => `99999999999999999999.99999999`
* Min: `(-10^28 + 1) / 10^8` => `-99999999999999999999.99999999`

I've also confirmed that the decimal equivalents of the scientific notation in the spec are correct.  So... we need to fix the values here in the code as well as in the tests.

Reply:
I have to wonder how we got all of these different representations all over! I'll fix it now.